contracts/src/TokenHome/TokenHome.sol

```diff
@@ -445,6 +480,7 @@ abstract contract TokenHome is ITokenHome, TeleporterOwnerUpgradeable, SendReent
         address originSenderAddress,
         bytes memory message
     ) internal override {
+        TokenHomeStorage storage $ = _getTokenHomeStorage();
```

Comment:
How did you decide when to call `_getTokenHomeStorage` and when to call the getter directly? Was it in instances where the contract state is referenced more than once (as is the case here)?

Reply:
For the getter functions for the struct storage type, they're all made private and only callable in defined contracts. Getters for the variables are used for derived contracts or external calls.